_monitoring-your-cluster/pa/reference.md

```diff
@@ -651,6 +651,140 @@ This list is extensive. We recommend using Ctrl/Cmd + F to find what you're look
       <td>FollowerCheck_Latency (WIP)
       </td>
     </tr>
+    <tr>
+      <td>searchbp_shard_stats_cancellationCount
+      </td>
+      <td rowspan="22">NodeID, searchbp_mode
+      </td>
+      <td>The number of tasks marked for cancellation, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_limitReachedCount
+      </td>
+      <td>The number of times when the number of tasks eligible for cancellation exceeded the set cancellation threshold, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_resource_heap_usage_cancellationCount
+      </td>
+      <td>The number of tasks marked for cancellation because of excessive heap usage since the node last restarted, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_resource_heap_usage_currentMax
+      </td>
+      <td>The maximum heap usage for tasks currently running, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_resource_heap_usage_rollingAvg
+      </td>
+      <td> The rolling average heap usage for n most recent tasks, on search shard task level. The default value for n is 100.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_resource_cpu_usage_cancellationCount
+      </td>
+      <td>he number of tasks marked for cancellation because of excessive CPU usage since the node last restarted, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_resource_cpu_usage_currentMax
+      </td>
+      <td>The maximum CPU time for all tasks currently running on the node, on search shard task level.
```

Comment:
`CPU time`, Is this `time` or `usage` ?

Reply:
I was referring to this [page](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/search-backpressure/) when writing. It has CPU time on it. 
<img width="932" alt="Screenshot 2023-10-13 at 12 56 16 PM" src="https://github.com/opensearch-project/documentation-website/assets/67210127/fe974108-f21d-4d59-b150-bf73456b948a">

Reply:
It is time in nanoseconds
```
public TaskResourceUsageTracker.Stats stats(List<? extends Task> activeTasks) {
        long currentMax = activeTasks.stream().mapToLong(t -> t.getTotalResourceStats().getCpuTimeInNanos()).max().orElse(0);
        long currentAvg = (long) activeTasks.stream().mapToLong(t -> t.getTotalResourceStats().getCpuTimeInNanos()).average().orElse(0);
        return new Stats(getCancellations(), currentMax, currentAvg);
    }
```